pkg/pod/status_test.go

```diff
@@ -646,40 +646,6 @@ func TestMakeTaskRunStatus(t *testing.T) {
 				}},
 			},
 		},
-	}, {
-		desc: "image resource updated",
```

Comment:
I wonder if it would make sense to keep this test case, but just have it not populate anything in resourcesresult?

Reply:
Thanks Lee. restored.